python_modules/dagster/dagster/core/instance/config.py

```diff
@@ -177,4 +253,5 @@ def dagster_instance_config_schema():
         "code_servers": Field(
             {"local_startup_timeout": Field(int, is_required=False)}, is_required=False
         ),
+        "tick_retention": tick_retention_config_schema(),
```

Comment:
do you think something like this could make sense to colocate with event log retention or run retention once that is available? Maybe this should just be 'retention' for future retention-related settings?